### PR TITLE
Add spaCy 1.1.0 and related dependencies

### PR DESCRIPTION
**Pull request by rolando**
Notes:

* Some recipes are missing `about/license_file` entry because the
source distribution itself doesn't include the license file.
* Even though some of theses packages exist in anaconda channel, there
is no win-* platform packages.

### CODE REVIEW

**Comment by rolando** [comment]
ping @conda-forge/core

File: recipes/thinc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    - numpy x.x
+    - murmurhash >=0.26,<0.27
+    - cymem >=1.30,<1.32
+    - preshed >=0.46,<0.47
+
+  run:
+    - python
+    - numpy x.x
+    - murmurhash >=0.26,<0.27
+    - cymem >=1.30,<1.32
```
Comment (ocefpaf) [inline recipes/thinc/meta.yaml]: Sending multiple recipes to staged-recipes makes it hard to review and, once merged, there will be a race condition where a dependency, `cymem` for example, won't be available when this recipe is first built on the feedstock. So you will need to watch the builds and re-start them accordingly to the availability of the dependencies.

Comment (rolando) [inline recipes/thinc/meta.yaml]: I see. I don't mind watching for packages being built. Is that a hard rule? There is a related PR #2455 that also includes a bunch of dependencies. I estimate it takes a minimum 3-5 days to get a recipe merged and the feedstock created. So doing a separate PR for each small dependency would imply a longer wait time until we get the final package published.

Comment (ocefpaf) [inline recipes/thinc/meta.yaml]: > @ocefpaf I see. I don't mind watching for packages being built. Is that a hard rule?

Not a hard rule, that is why I merged it.

> There is a related PR #2455 that also includes a bunch of dependencies. I estimate it takes a minimum 3-5 days to get a recipe merged and the feedstock created.

Maybe b/c it is a bunch of recipes :smile: I rarely review many recipes. This batch you were in luck b/c I am waiting in line with nothing to do :wink: 

> So doing a separate PR for each small dependency would imply a longer wait time until we get the final package published.

I understand, but that is recommended.

Comment (rolando) [inline recipes/thinc/meta.yaml]: Thanks @ocefpaf !

**Pull request merged**